OamLog+services.yaml

```diff
@@ -90,16 +119,19 @@ clients:
       application-name: NewRelease
       release-number: 1.0.0
       uuid: ol-1-0-0-http-c-0010
+            ol-1-0-0-http-c-im-nr-1-0-0-000
     tcp-client:
       ip-address: 10.118.125.157
       tcp-port: 1003
       uuid: ol-1-0-0-tcp-c-0010
+            ol-1-0-0-tcp-c-im-nr-1-0-0-000
     operation-clients:
       own-oam:
         basic:
         individual:
           - service-name: /v1/regard-application
             uuid: ol-1-0-0-op-c-1010
+                  ol-1-0-0-op-c-im-ne-1-0-0-000
```

Comment:
A small typo mistake here in uuid ol-1-0-0-op-c-im-ne-1-0-0-000 . application-id to be replaced with nr instead of ne